src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -348,7 +375,7 @@ def process_messages(
 
     for message in outer_message.payload:
         parsed_payload_value = parsed_payloads_by_offset[message.offset]
-        new_payload_value = parsed_payload_value
+        new_payload_value = parsed_payload_value.copy()
```

Comment:
I realized this was mutating the original message so if it got processed again it was already translated (bad)

Reply:
This is probably a deepcopy.